addons/account/views/account_move_views.xml

```diff
@@ -529,8 +529,7 @@
                                 type="object"
                                 string="Send &amp; Print"
                                 attrs="{'invisible':['|', '|', ('state', '!=', 'posted'), ('is_move_sent', '=', True), ('move_type', 'not in', ('out_invoice', 'out_refund'))]}"
-                                class="oe_highlight"
-                                groups="account.group_account_invoice"/>
+                                class="oe_highlight"/>
```

Comment:
You probably want to add the right `sale` group instead for this action. `sales_team.group_sale_salesman` ?

Reply:
The group will also need access to `model_account_invoice_send`, I don't think it is the case right now